### PR TITLE
added http.method to plugins

### PR DESCRIPTION
**Pull request by tom-pytel**

### CODE REVIEW

**Comment by tom-pytel** [comment]
Ok, I give up, what is failing?

**Comment by tom-pytel** [comment]
Hmm, was just working on this actually, but lets see how yours works.

**Comment by kezhenxu94** [comment]
> Hmm, was just working on this actually, but lets see how yours works.

Sorry for not letting you know and maybe cause some duplicate works, just got some time to check this minutes ago

**Review by kezhenxu94 — APPROVED** [review]

File: src/plugins/HttpPlugin.ts [hunk 1]
```diff
@@ -150,10 +154,16 @@ class HttpPlugin implements SwPlugin {
         try {
           span.component = Component.HTTP_SERVER;
           span.layer = SpanLayer.HTTP;
-          span.peer = req.connection.remoteFamily === 'IPv6'
-            ? `[${req.connection.remoteAddress}]:${req.connection.remotePort}`
-            : `${req.connection.remoteAddress}:${req.connection.remotePort}`;
+          // span.peer = req.connection.remoteFamily === 'IPv6'
+          //   ? `[${req.connection.remoteAddress}]:${req.connection.remotePort}`
+          //   : `${req.connection.remoteAddress}:${req.connection.remotePort}`;
```
Comment (kezhenxu94) [inline src/plugins/HttpPlugin.ts:159]: Can yo remove these if they're not used anymore

**Review by kezhenxu94 — APPROVED** [review]

**Pull request merged**